lib/descriptor-stream.js

```diff
@@ -1,20 +1,10 @@
 /* @flow */
-"use strict";
+'use strict';
 
-import EventEmitter from 'events';
+import {EventEmitter} from './events-flowtype-bug';
```

Comment:
It's an (old) bug in flow that prevents you to return a Promise with an object that directly extends EventEmitter.

Way to overcome this flow bug is to create another layer of redirection as seen here.

https://github.com/facebook/flow/issues/545